cmd/ebrelayer/txs/parser_test.go

```diff
@@ -128,46 +130,6 @@ func TestFailedLockEventToCosmosMsg(t *testing.T) {
 	require.Error(t, err)
 }
 
-func TestMsgBurnToProphecyClaim(t *testing.T) {
```

Comment:
I pulled in the test changes from feature/peggy42, and in that branch ethbridge.ProphecyClaim has been removed, so the tests were also removed.